### PR TITLE
Note that CellAssign could be labeled with `Not run`

### PR DESCRIPTION
**Pull request by allyhawkins**
In https://github.com/AlexsLemonade/scpca-nf/pull/662, we accounted for processed objects that do not successfully complete CellAssign due to low numbers of cells. There, we still include the `cellassign_celltype_annotation` column, but we populate it with `Not run`. I think I noted both areas where this column is mentioned to include the `Not run` option.

### CODE REVIEW

**Review by sjspielman — COMMENTED** [review]
I wanted to also indicate something about `cellassign_max_prediction`. What do you think of the wording I suggested? I'm not tied to it at all if you have other phrasing thoughts!

File: docs/merged_objects.md [hunk 1]
```diff
@@ -165,11 +165,11 @@ Each such list will contain the following fields:
...
+| `cellassign_predictions` | If cell typing with `CellAssign` was performed and completed successfully, the full matrix of predictions across cells and cell types |
+| `cellassign_reference` | If cell typing with `CellAssign` was performed and completed successfully, the reference name as established by the Data Lab used for cell type annotation |
+| `cellassign_reference_organs` | If cell typing with `CellAssign` was performed and completed successfully, a comma-separated list of organs and/or tissue compartments from which marker genes were obtained to create the reference |
+| `cellassign_reference_source`  | If cell typing with `CellAssign` was performed and completed successfully, the source of the reference dataset (default is [`PanglaoDB`](https://panglaodb.se/)) |
+| `cellassign_reference_version`  | If cell typing with `CellAssign` was performed and completed successfully, the version of the reference dataset source. For references obtained from `PanglaoDB`, the version scheme is a date in ISO8601 format |
```
Comment (sjspielman) [inline docs/merged_objects.md:172]: 🐱

**Comment by allyhawkins** [comment]
@sjspielman I like the wording you suggested for the max prediction score. I also used that same wording for any of the metadata slots related to `CellAssign` to indicate they are present if `CellAssign was performed and completed successfully`.

**Review by sjspielman — APPROVED** [review]
🐶

**Pull request merged**